File: 0268-missing-number/0268-missing-number.py
class Solution(object):
    def missingNumber(self, nums):

        # for i in range(len(nums)+1):
        #     if i not in nums:
        #         return i
        
        n = len(nums)

        sum_nums = (n) * (n+1) // 2

        curr = sum(nums)
        return  sum_nums - curr
        
    

File: 0268-missing-number/0268-missing-number.js
/**
 * @param {number[]} nums
 * @return {number}
 */
var missingNumber = function(nums) {
    num_len=nums.length
// console.log(num_len)



    for(i=0;i<=num_len;i++){
        if (!nums.includes(i)){
            return i
        }
    }
    }    
